frontend_app_agenda/package.json

```diff
@@ -4,12 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "servdev": "NODE_ENV=development webpack-dev-server --watch --colors --inline --hot --progress",
-    "servdevwindoz": "set NODE_ENV=development&& webpack-dev-server --watch --colors --inline --hot --progress",
+    "servdev": "NODE_ENV=development webpack-dev-server --config webpack.tracim.config.js --watch --colors --inline --hot --progress",
+    "servdevwindoz": "NODE_ENV=development webpack-dev-server --config webpack.tracim.config.js --watch --colors --inline --hot --progress",
     "build": "NODE_ENV=production webpack-cli -p",
+    "tracimbuild": "NODE_ENV=production webpack-cli -p --config webpack.tracim.config.js",
     "build-dev": "NODE_ENV=production webpack-cli",
-    "buildwindoz": "set NODE_ENV=production&& webpack-cli -p",
-    "build-devwindoz": "set NODE_ENV=production&& webpack-cli",
+    "tracimbuild-dev": "NODE_ENV=production webpack-cli --config webpack.tracim.config.js",
+    "buildwindoz": "build",
+    "tracimbuildwindoz": "tracimbuild",
+    "build-devwindoz": "build-dev",
+    "tracimbuild-devwindoz": "tracimbuild-dev",
```

Comment:
I don't think any of this will work
On windows, command lines needs a slightly different syntax

Reply:
I guess this comment it outdated since your response to the question about windows command lines